demos/super_resolution_demo/README.md

```diff
@@ -4,15 +4,19 @@ This topic demonstrates how to run Super Resolution demo application, which
 reconstructs the high resolution image from the original low resolution one.
 You can use the following pre-trained model with the demo:
 
+* `single-image-super-resolution-1032`, which is the primary and only model that
+  performs super resolution 4x upscale on a 270x480 image
 * `single-image-super-resolution-1033`, which is the primary and only model that
-  performs super resolution 4x upscale on a 200x200 image
+  performs super resolution 3x upscale on a 360x640 image
+* `text-image-super-resolution-0001`, which is the primary and only model that
```

Comment:
Do you not think that 2 models claimed as ` the primary and only model` with the same image resolution and upscale look slightly strange?